setup.cfg

```diff
@@ -23,7 +23,7 @@ install_requires =
   python-dateutil
   absl-py
   jinja2
-  pandas
+  tabulate
```

Comment:
Can you verify if numpy is used anywhere else aswell :D

Reply:
I just tried with no numpy or pandas and it works fine.